devicetypes/Orvibo/Orvibo-Motion-Sensor.src/Orvibo-Motion-Sensor.groovy

```diff
@@ -0,0 +1,126 @@
+/**
+ *  Orvibo Motion Sensor
+ *
+ *  Copyright 2016 Chris Kitch
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "Orvibo Motion Sensor", namespace: "Orvibo", author: "jk0218.zhang@samsung.com", vid: "generic-motion-2", ocfDeviceType: "x.com.st.d.sensor.motion") {
+		capability "Motion Sensor"
+		capability "Battery"
+
+        fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001", outClusters: "", model: "895a2d80097f4ae2b2d40500d5e03dcc", manufacturer: "ORVIBO"
+	}
+    
+	preferences {
+		section {
+			input title: "Motion Timeout", description: "These devices don't report when motion stops, so it's necessary to have a timer to report that motion has stopped. You can adjust how long this is below.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+			input "motionStopTime", "number", title: "Seconds", range: "*..*", displayDuringSetup: false, defaultValue: 5
+		}
+	}
+}
+
+// Parse incoming device messages to generate events
+def parse(String description) {
+    def name = null
+	def value = description
+	def descriptionText = null
+    log.debug "Parsing: ${description}"
+    
+	Map map = [:]
+
+	List listMap = []
+	List listResult = []
+    
+	if (zigbee.isZoneType19(description)) {
+		name = "motion"
+		def isActive = zigbee.translateStatusZoneType19(description)
+        
+		value = isActive ? "active" : "inactive"
+		descriptionText = getDescriptionText(isActive)
+	} else if(description?.startsWith("read attr -")) 
+    	map = parseReportAttributeMessage(description)
+
+	if (value == "active") {
+    	def timeout = 2
+        
+        if (motionStopTime)
```

Comment:
You will need to put in extra code here to use the default value. The "default" in the preference definition does not actually set the value in memory.